src/plugins/plugin.tooltip.js

```diff
@@ -870,14 +887,14 @@ export class Tooltip extends Element {
 
   draw(ctx) {
     const me = this;
-    const options = me.options;
+    const options = me.options.setContext(me.getContext());
```

Comment:
I think this could be avoided if the `me.options` was replaced in `update`. Not probably a big performance hit, but still done in every frame.